docs/examples/extension.ipynb

```diff
@@ -11,14 +11,14 @@
    "cell_type": "markdown",
```

Comment:
<p>This looks good, but there's a few formatting issues. Remove excess vertical white space, and make sure the docstring has triple-quotes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/halomod/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>not sure if I did correctly</p>